wagn/lib/wagn/parser.rb

```diff
@@ -0,0 +1,93 @@
+# -*- encoding : utf-8 -*-
+
+module Wagn
+  class Parser
+    class << self
+      def wagn envs
+        OptionParser.new do |parser|
+          parser.banner = "Usage: wagn #{command} [options]\n\n" \
+                          "Run wagn:#{command} task on the production "\
+                          " database specified in config/database.yml\n\n"
+          parser.on('--production', '-p',
+                    "#{command} production database (default)") do
+            envs = ['production']
+          end
+          parser.on('--test', '-t',
+                    "#{command} test database") do
+            envs = ['test']
+          end
+          parser.on('--development', '-d',
+                    "#{command} development database") do
+            envs = ['development']
+          end
+          parser.on('--all', '-a',
+                    "#{command} production, test, and development database") do
+            envs = %w( production development test)
+          end
+        end
+      end
+
+      def rspec opts
```

Comment:
Assignment Branch Condition size for rspec is too high. [36.89/15]<br>Method has too many lines. [55/15]